frontend/public/components/autocomplete.tsx

```diff
@@ -56,13 +55,13 @@ const AutocompleteInput: React.FC<AutocompleteInputProps> = (props) => {
 
   React.useEffect(() => {
     if (textValue && visible && showSuggestions) {
-      const processed = labelParser(data);
+      const processed = labelParser(data, props.labelPath);
```

Comment:
Nit: Define `labelPath` instead of using `props.labelPath` to be consistent with how this component uses other props.